ccvm_simulators/solvers/ccvm_solver.py

```diff
@@ -338,3 +348,150 @@ def machine_energy(self, machine: str, machine_parameters: dict = None):
             )
 
         return energy_method(machine_parameters)
+
+    ##############################
+    ### MACHINE TIME FUNCTIONS ###
+    ##############################
+
+    def _validate_machine_time_dataframe_columns(self, dataframe):
```

Comment:
I've noticed that this function shares identical logic with the one present in the machine energy functions. Do you think it's preferable to keep them separate, or would it also make sense to merge them into a single function named `_validate_dataframe_columns`? Consolidating them could reduce redundancy. What are your thoughts on this approach?

Reply:
I think @ChanWooY-1QBit  made a good point about things being more pythonic if we use a try block instead of a validator here (I'm paraphrasing a bit). 
Having validation on the machine parameters makes sense due to way we use the wrapper around the function, but since this is a private function I think try-except would be cleaner. It'll sacrifice some of the details we are giving the user in the error message, but we can leave it up to the user to check their own data. I'll remove this function completely, but leave the energy version since it's not related to this PR.

Reply:
Thanks for explaining your thoughts. Sounds good to me.